src/azul/transformer.py

```diff
@@ -635,6 +635,26 @@ def get(self):
         return self.value.get()
 
 
+class SetCountingAccumulator(Accumulator):
+    """
+    Count the number of unique values
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.set_accum = SetAccumulator()
```

Comment:
```suggestion
        self.value = SetAccumulator()
```

For consistency with other accumulators.